santiao-guillen/homework-3/spongebobMeme.js

```diff
@@ -0,0 +1,14 @@
+function spongeMeme(sentence) {
+    let array = [];
+    let result;
+
+    for (const value in sentence) {
```

Comment:
@santiagoGuillenGlob , use modern approaches for array operations like forEach, map, reduce.